app/controllers/schools_controller.rb

```diff
@@ -12,11 +12,11 @@ def show
   end
 
   def search
-    School.all.collect { |school| ["#{school.name}, #{school.city}, #{school.state}", school.name] }
+    School.all.collect { |school| ["#{school.name}, #{school.country}, #{school.city}, #{school.state}", school.name] }
   end
 
   def create
-    @school = School.find_by(name: school_params[:name], city: school_params[:city], state: school_params[:state])
+    @school = School.find_by(name: school_params[:name], country: school_params[:country], city: school_params[:city], state: school_params[:state])
```

Comment:
This is starting to look / read a little unwieldy.

I might write a method called `unique_school_params` or something, which then just permit's only the valid list of parameters. (You can have multiple method which do parameter filtering)

```suggestion
    @school = School.find_by(unique_school_params)
```